backend/src/models/__tests__/game.model.test.ts

```diff
@@ -146,6 +147,25 @@ describe("Game Model", () => {
     expect(savedGame.players[2].punchlines).toHaveProperty("length", 0);
     expect([...savedGame.players]).toMatchObject(Array(3).fill({ new: true }));
     expect([...savedGame.players]).toMatchObject(Array(3).fill({ score: 0 }));
+    expect(validateUUID(savedGame.players[0].token)).toBeTruthy();
+    expect(validateUUID(savedGame.players[1].token)).toBeTruthy();
+    expect(validateUUID(savedGame.players[2].token)).toBeTruthy();
+  });
+
+  it("throws a ValidationError if a player is added without a nickname", async () => {
+    gameData.players = [
+      { punchlines: ["To get to the other side"] },
+      { punchlines: ["To go to KFC"] },
+    ];
+
+    const game = new GameModel(gameData);
+
+    try {
+      await game.save();
```

Comment:
should put a fail() in here to make sure it throws an error

Reply:
Yep good catch!